pkg/plugins/manager/installer/logger/logger.go

```diff
@@ -0,0 +1,56 @@
+package logger
+
+import (
+	"fmt"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+)
+
+type Logger interface {
```

Comment:
Nit. This could perhaps live side-by-side with the `Installer` interface and named `InstallerLogger`? The InfraLogWrapper could perhaps be an internal implementation in the manager package.